.github/actions/build/action.yml.j2

```diff
@@ -36,7 +39,7 @@ runs:
         target-device: "${{ inputs.target-device }}"
         git_sha: "<<package_info.git_tag>>"
         host-platform: ${{ inputs.host-platform }}
-        dest-dir: ${{ env.ARTIFACTS_DIR }}
+        dest-dir: "${{ env.REPO_DIR }}/<<package_info.artifact_name>>"
```

Comment:
We can remove all code related to download-artifacts. I have created a PR based on this PR with the required changes: https://github.com/NVIDIA/cuda-python/pull/250. Please merge the aforementioned PR with this PR. Though you may need to rebase this PR on main before you do that.